src/components/Tooltip.tsx

```diff
@@ -1,32 +1,36 @@
+/* eslint-disable no-case-declarations */
```

Comment:
Remove or add justification

Reply:
eslint does not allow declarations in switch, which is annoying